src/compute-types/src/explain/text.rs

```diff
@@ -849,3 +866,29 @@ impl<'a> fmt::Display for Permutation<'a> {
         }
     }
 }
+
+/// Annotations for physical plans.
+struct PlanAnnotations {
+    config: ExplainConfig,
+    node_id: NodeId,
+}
+
+impl PlanAnnotations {
```

Comment:
```suggestion
// The current implementation deviates from the `AnnotatedPlan` used in
// `Mir~`-based plans. This is fine, since at the moment we the only attribute
// we are going to explain is the `node_id`, which at the moment is kept inline
// with the `Plan` variants. If at some point in the future we want to start
// deriving and printing attributes that are derived ad-hoc, however, we might
// want to adopt `AnnotatedPlan` here as well.
impl PlanAnnotations {
```

Reply:
Thanks!